sdk/cosmosdb/cosmos/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@azure/cosmos",
+  "description": "Azure Cosmos DB Service Node.js SDK for SQL API",
+  "keywords": [
+    "cosmosdb",
+    "cosmos db",
+    "documentdb",
+    "document database",
+    "azure",
+    "nosql",
+    "database",
+    "cloud"
+  ],
+  "version": "2.1.5",
+  "author": "Microsoft Corporation",
+  "main": "./lib/src/index.js",
+  "types": "./lib/src/index.d.ts",
+  "engine": {
+    "node": ">=6.0.0"
+  },
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "clean": "rimraf lib",
+    "lint": "tslint --project tsconfig.json",
+    "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\"",
+    "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\"",
+    "compile": "echo Using TypeScript && tsc --version && tsc --pretty",
+    "compile-prod": "echo Using TypeScript && tsc --version && tsc -p tsconfig.prod.json --pretty",
+    "prepack": "npm install && npm run build",
+    "pack": "npm pack 2>&1",
+    "webpack": "webpack -d",
+    "webpack-prod": "webpack -p",
+    "build": "npm run clean && npm run check-format && npm run lint && npm run compile && npm run webpack",
+    "build-prod": "npm run clean && npm run check-format && npm run lint && npm run compile-prod && npm run webpack-prod",
```

Comment:
Recommend you switch around the convention used here. The build system will run `build` to produce production code. For inner-loop development, it's probably best to name them `compile-dev` / `build-dev`/ `webpack-dev` or something like that.

Reply:
It may disrupt developers' normal workflow.  Could the build system use `build-prod`, fall back to `build` if there's no `build-prod`)?

Reply:
I'd have to add "build-prod" tasks to every other package.json, which I'd like to avoid. Rush runs scripts for many packages in parallel, so it assumes that each package is laid out basically the same from the standpoint of what scripts it defines.

I'm very much in favor of not disrupting devs' existing workflows, but cosmosdb appears to be the only package that uses this convention. Additionally, if we want to continue to allow customers to install our packages direct from the Git repo, `npm run build` will be invoked automatically on the customer's machine, and we want the customer to receive a production build artifact.

Reply:
`build` for prod is fine and makes sense.

Reply:
I swap the two. Non-production scripts now have `-dev` suffix.